[Bug] Github Action fails unexpectedly on build-and-test (macOS-latest)

## Bug description
Github action "build-and-test macOS-latest" fails unexpectedly, while windows and ubuntu succeed. The provided error below was during a readme change and can therefore not be related to code changes.
> error[E0463]: can't find crate for `serde_derive` which `serde_json` depends on

[Link to an example error](https://github.com/iotaledger/identity.rs/pull/164/checks?check_run_id=2191771082).

It may be related to [this issue](https://github.com/rust-lang/cargo/issues/8603) which triggered on the same environment and even the same crate `serde_derive` and [this issue](https://github.com/actions/virtual-environments/issues/2619). 

Potential workaround: Disable cache. 

## Errors
> error[E0463]: can't find crate for `serde_derive` which `serde_json` depends on
Error:  --> identity-diff/src/object.rs:4:5
  |
4 | use serde_json::Value;
  |     ^^^^^^^^^^ can't find crate

--- a/.github/workflows/build-and-test.yml
+++ b/.github/workflows/build-and-test.yml
@@ -38,7 +38,7 @@ jobs:
         run: echo "CURRENT_DATE=$(Get-Date -Format "yyyy-MM-dd")" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
 
       - name: Cache cargo registry
-        uses: actions/cache@v2
+        uses: actions/cache@v2.1.4
         with:
           path: ~/.cargo/registry
           # Add date to the cache to keep it up to date
@@ -48,7 +48,7 @@ jobs:
             ${{ matrix.os }}-cargo-registry-${{ hashFiles('**/Cargo.toml') }}
 
       - name: Cache cargo index
-        uses: actions/cache@v2
+        uses: actions/cache@v2.1.4
         with:
           path: ~/.cargo/git
           # Add date to the cache to keep it up to date
@@ -58,7 +58,7 @@ jobs:
             ${{ matrix.os }}-cargo-index-${{ hashFiles('**/Cargo.toml') }}
 
       - name: Cache cargo target
-        uses: actions/cache@v2
+        uses: actions/cache@v2.1.4
         with:
           path: target
           # Add date to the cache to keep it up to date
@@ -68,7 +68,7 @@ jobs:
             ${{ matrix.os }}-cargo-build-target-${{ hashFiles('**/Cargo.toml') }}
 
       - name: Cache wasm cargo target
-        uses: actions/cache@v2
+        uses: actions/cache@v2.1.4
         with:
           path: bindings/wasm/target
           # Add date to the cache to keep it up to date

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -7,7 +7,6 @@ members = [
   "identity-did",
   "identity-diff",
   "identity-iota",
-
   "examples",
 ]
 

